login-mobile.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>Locaholic-mobile view</title>

```

Comment:
change the title to Locaholic|Some tag line  (ask @Prashantkankaria  for this tagline)